Remove use of composer.phar in CI

Something in user_ldap CI is trying to:
```
+ cd /var/www/owncloud/apps/user_ldap
+ make
mkdir /var/www/owncloud/apps/user_ldap/build
cd /var/www/owncloud/apps/user_ldap/build && curl -sS https://getcomposer.org/installer | php
curl: (7) Failed to connect to getcomposer.org port 443: Connection reset by peer
php /var/www/owncloud/apps/user_ldap/build/composer.phar install --dev
Could not open input file: /var/www/owncloud/apps/user_ldap/build/composer.phar
make: *** [lib/composer/phpunit] Error 1
Makefile:76: recipe for target 'lib/composer/phpunit' failed 
```
https://drone.owncloud.com/owncloud/user_ldap/889/335

- this fails "randomly" in CI when trying to get composer online
- anyway we don't want to do that, ``composer`` should already be there available

--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,10 @@
 SHELL := /bin/bash
 
+COMPOSER_BIN := $(shell command -v composer 2> /dev/null)
+ifndef COMPOSER_BIN
+    $(error composer is not available on your system, please install composer)
+endif
+
 #
 # Define NPM and check if it is available on the system.
 #
@@ -25,8 +30,6 @@ ldap_src_dirs=appinfo css img js lib l10n templates vendor
 ldap_all_src=$(ldap_src_dirs) $(ldap_doc_files)
 build_dir=$(CURDIR)/build
 dist_dir=$(build_dir)/dist
-COMPOSER_BIN=$(build_dir)/composer.phar
-PHAN_BIN=$(build_dir)/phan.phar
 
 # internal aliases
 composer_deps=vendor/
@@ -157,7 +160,6 @@ test-php-unit: $(composer_dev_deps)
 test-php-unit-dbg: ## Run php unit tests using phpdbg
 test-php-unit-dbg: $(composer_dev_deps)
 	$(PHPUNITDBG) --configuration ./phpunit.xml --testsuite unit
-
 
 .PHONY: test-acceptance-api
 test-acceptance-api: ## Run API acceptance tests

